gsw/stability.py

```diff
@@ -47,7 +47,7 @@ def Nsquared(SA, CT, p, lat=None, axis=0):
     Returns
     -------
     N2 : array
-        Buoyancy frequency-squared at pressure midpoints, 1/s.
+        Buoyancy frequency-squared at pressure midpoints, 1/s^2.
```

Comment:
I wonder if we should add the same text from the docs in http://www.teos-10.org/pubs/gsw/html/gsw_Nsquared.html

```
The units of N2 are radians2 s-2 however in may textbooks this is
 abreviated to s-2 as radians does not have a unit.  To convert the
 frequency to hertz, cycles sec-1, divide the frequency by 2π, ie N/(2π).
```

Reply:
Seems a good idea, does the license authorizes it? If so I'll be happy to commit this

Reply:
Our current docstrings are extracted from the matlab ones, used for that page. I'm not sure why this one mismatches and it is hard to follow the matlab code b/c it is not versioned.

TL;DR I believe it is OK. What do you think @efiring?